processor/memorylimiterprocessor/memorylimiter.go

```diff
@@ -5,83 +5,27 @@ package memorylimiterprocessor // import "go.opentelemetry.io/collector/processo
 
 import (
 	"context"
-	"errors"
-	"fmt"
-	"runtime"
-	"sync"
-	"sync/atomic"
-	"time"
-
-	"go.uber.org/zap"
 
 	"go.opentelemetry.io/collector/component"
-	"go.opentelemetry.io/collector/internal/iruntime"
+	memorylimiter "go.opentelemetry.io/collector/internal/memoryLimiter"
```

Comment:
```suggestion
	"go.opentelemetry.io/collector/internal/memorylimiter"
```

Reply:
lint fails when i dont do that. originally, it was just `"go.opentelemetry.io/collector/internal/memorylimiter"`

```
./opentelemetry-collector/.tools/golangci-lint run
factory.go:12: File is not `goimports`-ed with -local go.opentelemetry.io/collector (goimports)
        "go.opentelemetry.io/collector/internal/memoryLimiter"
memorylimiter.go:10: File is not `goimports`-ed with -local go.opentelemetry.io/collector (goimports)
        "go.opentelemetry.io/collector/internal/memoryLimiter"
factory_test.go:16: File is not `goimports`-ed with -local go.opentelemetry.io/collector (goimports)
        "go.opentelemetry.io/collector/internal/memoryLimiter"
```

Reply:
Why `memoryLimiter` instead of `memorylimiter`?

Reply:
oh i see, i was confused when golint requested this